### PR TITLE
Document the available database migration tools.

### LINKED ISSUE #46: Migrations
I started working on a Migrations DSL, with G-d's help. I posted the initial code here: https://groups.google.com/d/topic/scalaquery/_Kk_hrkliCM/discussion

### PR DESCRIPTION
**Pull request by lastland**
Resolve #46.

Reviewed by @cvogt @nafg ?

### CODE REVIEW

**Comment by typesafehub-validator** [comment]
Hi @lastland,

Thank you for your contribution! We really value the time you've taken to put this together.

Before we proceed with reviewing this pull request, please sign the Typesafe Contributors License Agreement:

[http://www.lightbend.com/contribute/cla](http://www.lightbend.com/contribute/cla)

**Comment by d6y** [comment]
This is super useful - thank you. 

I'm wondering: rather than put this in the Slick Manual, it might be better in the third-party documentation: (e.g., [this repository](https://github.com/slick/slick.github.com/tree/master/docs) which is used to generate e.g., http://slick.lightbend.com/docs/third-party-slick-3 and http://slick.lightbend.com/docs/third-party-slick-2)

File: slick/src/sphinx/migrations.rst [hunk 1]
```diff
@@ -0,0 +1,26 @@
+Database Migrations
+===========================
+Slick itself does not support database migrations, but there are some
+third-party tools that work well with Slick. This page provides a list
+of them.
+
+Scala Forklift
+---------------------------
+`Scala Forklift <https://github.com/lastland/scala-forklift>`_ is a
+type-safe data migration tool with great supports for Slick. Users can
```
Comment (cvogt) [inline slick/src/sphinx/migrations.rst]: s/supports/support/

File: slick/src/sphinx/migrations.rst [hunk 2]
```diff
@@ -0,0 +1,26 @@
+Database Migrations
+===========================
+Slick itself does not support database migrations, but there are some
```
Comment (cvogt) [inline slick/src/sphinx/migrations.rst]: s/not support/not have out of the box support/

**Comment by cvogt** [comment]
@d6y a reason for putting it into the docs would be that it is a quite important topic where people would be looking in the docs.

**Comment by d6y** [comment]
@cvogt makes sense - thanks.

**Comment by lastland** [comment]
I have made some modifications according to @cvogt 's suggestions.

**Comment by d6y** [comment]
Looks good to me:

database migrations slick 3 2 0-snapshot documentation 2016-05-27 10-45-19

**Comment by cvogt** [comment]
LGTM

**Comment by cvogt** [comment]
thank you for your screen shots @d6y always very helpful :)

**Comment by cvogt** [comment]
nice work @lastland

**Pull request merged**